src/prefect/events/utilities.py

```diff
@@ -6,7 +6,9 @@
 from uuid import UUID
 
 import prefect.types._datetime
+from prefect.exceptions import EventTooLarge
 from prefect.logging.loggers import get_logger
+from prefect.settings import PREFECT_EVENTS_MAXIMUM_SIZE_BYTES
```

Comment:
can we use `get_current_settings` and dot access to the setting instead of the legacy env var name?

Reply:
yes can do!